src/plugin/helpers/prosemirror.ts

```diff
@@ -175,7 +177,7 @@ const removeNode = (getPos: () => number | undefined) => (
   dispatch: ((tr: Transaction) => void) | false,
   view?: EditorView
 ) => {
-  if (!dispatch) {
+  if (dispatch == false) {
```

Comment:
```suggestion
  if (dispatch === false) {
```

Reply:
Checked twice for `==` instead of `===`, but clearly one still got through. Thanks for spotting!